Tools/rocprof-compute/rocprof-compute_advanced.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# MIT License
+#
+# Copyright (c) 2025 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+EXAMPLE_TOOL="rocprof-compute"
+EXAMPLE_BIN="${EXAMPLE_TOOL}_vcopy"
+EXAMPLE_WORKLOAD="./$EXAMPLE_BIN -n 1048576 -b 256"
```

Comment:
```suggestion
EXAMPLE_BIN="${EXAMPLE_TOOL}_occupancy"
EXAMPLE_WORKLOAD="./$EXAMPLE_BIN"
```
I suggest we just move to occupancy.  I've decided that vcopy is just too "un-interesting".